source/release-notes/backend.txt

```diff
@@ -13,6 +13,16 @@ Atlas App Services Changelog
 2023 Releases
 -------------
 
+.. _backend_20230223:
+
+23 February 2023 Release
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Add additional ``currentPasswordValid`` parameter to :ref:`password reset function
+  <auth-run-a-password-reset-function>`.
+- Update Atlas Device Sync with Flexible Sync to :ref:`use the same rules and permissions
```

Comment:
note to reviewer: this ref broken until Rules v2 feature branch is merged (#324)